app/services/get-reductions-export.js

```diff
@@ -0,0 +1,19 @@
+const getBreadcrumbs = require('./get-breadcrumbs')
+const getOrganisationUnit = require('./helpers/org-unit-finder')
+const getReductionsData = require('./data/get-reduction-notes-export')
+
+module.exports = function (id, organisationLevel) {
+  var result = {}
+  var reductionPromise = {}
+  var organisationalUnitType = getOrganisationUnit('name', organisationLevel)
+
+  reductionPromise = getReductionsData(id, organisationLevel)
```

Comment:
This promise could be combined into line 13, i.e

`return getReductionsData(id, organisationLevel).then(function (results) {`

to keep the code a bit more concise